x-pack/packages/observability/observability_utils/observability_utils_browser/hooks/use_abortable_async.ts

```diff
@@ -43,6 +61,13 @@ export function useAbortableAsync<T>(
       setError(undefined);
     }
 
+    function handleError(err: Error) {
+      setError(err);
+      // setValue(undefined);
```

Comment:
nit: commented code, was it a mistake?

Reply:
thank you! it is, at least the fact that the comment is still there.